### DISCUSSION TITLE
Best option to install enmapbox on new Linux (Ubuntu) machine

### DISCUSSION THREAD

**Discussion by aloboa** [EnMAP-Box/enmap-box#923] [Announcements]
What are te advantages/disadvantages of the 2 options described in the doc?
Option A: Install QGIS via the official repository
Option B: Install QGIS via conda/mamba
In particular, to avoid troubles with enmapbox updates...

**Comment by jakimowb** [comment]
It depends on your personal flavor.
conda/mamba is probably easier to define multiple environments and switch between them.

In both cases you need to take care of package updates (`sudo apt update` or `conda update`)

If the EnMAP-Box code that is released passes the test pipelines (which unfortunatelly is [actually not the case](https://github.com/EnMAP-Box/enmap-box/commit/a74df160d92cf6bacb984cc2c4e75842c60b8b8c) ), it should always run with the latest ubuntu apt / conda environments as well.
